xc/device.ml

```diff
@@ -1968,7 +1968,7 @@ module Dm_Common = struct
     let stop_varstored () =
       Varstored.stop ~xs domid;
       let dbg = Printf.sprintf "stop domid %d" domid in
-      Xenops_sandbox.Varstore_guard.stop dbg ~domid
+      Xenops_sandbox.Varstore_guard.stop dbg ~domid ~vm_uuid:(Uuidm.to_string (Xenops_helpers.uuid_of_domid ~xs domid))
```

Comment:
This could've passed the Uuidm.t instead of the string, then you only need to do the conversion in one place, where it is used.

Reply:
and also it then makes the type of the function different from the others (Uuidm.t rather than string in one case)